src/katgpucbf/fgpu/send.py

```diff
@@ -252,14 +253,22 @@ def make_streams(
                 ttl=ttl,
                 comp_vector=comp_vector,
                 memory_regions=memory_regions,
+                buffer_size=buffer // len(interfaces),
             )
             for interface in interfaces
         ]
         streams = [spead2.send.asyncio.UdpIbvStream(thread_pool, config, ibv_config) for ibv_config in ibv_configs]
     else:
+        # TODO: the type: ignore is just until
+        # https://github.com/ska-sa/spead2/pull/282 is released.
         streams = [
-            spead2.send.asyncio.UdpStream(
-                thread_pool, [(ep.host, ep.port) for ep in endpoints], config, ttl=ttl, interface_address=interface
+            spead2.send.asyncio.UdpStream(  # type: ignore
+                thread_pool,
+                [(ep.host, ep.port) for ep in endpoints],
+                config,
+                ttl=ttl,
+                interface_address=interface,
+                buffer_size=buffer // len(interfaces),
```

Comment:
So the buffer is shared between interfaces? I guess in most cases `len(interfaces)` will be 1, but in the high-bandwidth case it may be 2.

Reply:
Yes. This makes it consistent with `--src-buffer`. On the other hand, I guess that means that with two interfaces, the buffer is smaller than it was. I think I'll just increase the default: the reason I added this support was there was some inconclusive evidence that increasing the buffer size might help, and I wanted to be able to test that easily.